packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -246,6 +250,17 @@ try {
 	const primaryPhp = await requestHandler.getPrimaryPhp();
 	await apiEndpoint.setPrimaryPHP(primaryPhp);
 
+	// NOTE: We need to derive the loaded WP version or we might assume WP loaded
+	// from browser storage is the default version when it is actually something else.
+	// Incorrectly assuming WP version can break things like remote asset retrieval
+	// for minified WP builds.
+	const loadedWordPressVersion = await getLoadedWordPressVersion(
+		requestHandler
+	);
+	if (isSupportedWordPressVersion(loadedWordPressVersion)) {
```

Comment:
With this check we won't always expose `apiEndpoint.wordPressVersion` which seems confusing. It makes it seem like we're not running any WordPress version if the version string doesn't match the pattern. What's an upside?

Reply:
Ah, this actually overrides the previous value of `wordPressVersion`. Still, can't we always override?

Reply:
> Ah, this actually overrides the previous value of wordPressVersion. Still, can't we always override?

@adamziel This is the version used to download static assets. I think we should make it so that we just don't attempt static asset downloads for unsupported versions but plan to do that in a follow up PR.

Reply:
I changed my mind and updated this PR to tolerate an undefined `staticAssetsDirectory` when loading an unsupported WP version. It didn't seem right to land this in an in-between state.

In addition, instead of clobbering the requested WordPress version with the detected version, I made them different properties:
- `requestedWordPressVersion`
- `loadedWordPressVersion`

Reply:
Good call!